lib/Element.js

```diff
@@ -99,7 +111,7 @@ Element.prototype.attributeToPropName = function (name) {
     return expr.test(name)
   })
 
-  if (some(skipTransformMatches)) {
+  if (skipTransformMatches.some(isTruthy)) {
```

Comment:
Ooh, shiny! :D

It'd be nice if I didn't have to pass an argument at all.

On 26 February 2016 at 13:51, Nicolas Brugneaux <notifications@github.com>
wrote:

> In lib/Element.js
> <https://github.com/Olical/react-faux-dom/pull/37#discussion_r54244392>:
>
> > @@ -99,7 +111,7 @@ Element.prototype.attributeToPropName = function (name) {
> >      return expr.test(name)
> >    })
> >
> > -  if (some(skipTransformMatches)) {
> > +  if (skipTransformMatches.some(isTruthy)) {
>
> skipTransformMatches.some(Boolean) would do the trick too [image:
> :guitar:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Olical/react-faux-dom/pull/37/files#r54244392>.
>